src/mc_rbdyn/configuration_io.cpp

```diff
@@ -149,9 +150,39 @@ mc_rtc::Configuration ConfigurationLoader<mc_rbdyn::JointSensor>::save(const mc_
 
 mc_rbdyn::Collision ConfigurationLoader<mc_rbdyn::Collision>::load(const mc_rtc::Configuration & config)
 {
-  return mc_rbdyn::Collision(config("body1"), config("body2"), config("iDist", 0.05), config("sDist", 0.01),
-                             config("damping", 0.0), config("r1Joints", std::vector<std::string>{}),
-                             config("r2Joints", std::vector<std::string>{}));
+  auto body1 = config("body1");
+  auto body2 = config("body2");
+  auto loadActiveJoints = [&](std::string prefix) -> std::tuple<std::optional<std::vector<std::string>>, bool>
+  {
+    if((config.has(prefix + "ActiveJoints") || config.has(prefix + "Joints")) && config.has(prefix + "InactiveJoints"))
+    {
+      mc_rtc::log::warning(
+          "Collision ({} - {}) has both {}ActiveJoints and {}InactiveJoints, {}ActiveJoints will be used", body1, body2,
+          prefix);
+    }
```

Comment:
This ends up doing the look up multiple times, I suggest:
```cpp
auto active_joints = config.find(prefix + "ActiveJoints");
auto joints = config.find(prefix + "Joints");
auto inactive_joints = config.find(prefix + "InactiveJoints");
if((active_joints || joints) && inactive_joints)
{
  mc_rtc::log::warning(
          "Collision ({} - {}) has both {}ActiveJoints and {}InactiveJoints, {}ActiveJoints will be used", body1, body2,
          prefix);
}
```

Then use the values below